src/app/pages/StakingPage/Features/ValidatorList/index.tsx

```diff
@@ -93,6 +99,9 @@ export const ValidatorList = memo((props: Props) => {
   return (
     <Box pad="medium" background="background-front">
       {t('common.validators', 'Validators')}
+      {updateValidatorsError ? (
+        <p>Couldn't load validators. List may be empty or out of date. {updateValidatorsError}</p>
```

Comment:
frontend devs please review

Reply:
I think this error should be translated using `t()`?

Reply:
ah yeah the outer part should

Reply:
added